other/drawer/src/draw.ts

```diff
@@ -0,0 +1,74 @@
+import { AptosAccount, AptosClient, BCS } from "aptos";
+import {
+  AccountAddress,
+  ChainId,
+  EntryFunction,
+  RGBAXY,
+  RawTransaction,
+  TransactionPayloadEntryFunction,
+} from "./canvas_type";
+import { API_GATEWAY_URL, API_TOKEN, CANVAS_CONTRACT_ADDRESS, GAS_LIMIT, GAS_PRICE } from "./const";
+import { serializeVectorU64, serializeVectorU8 } from "./util";
+
+const CLIENT = new AptosClient(API_GATEWAY_URL, {
+  HEADERS: {
+    authorization: `Bearer ${API_TOKEN}`,
+  },
+});
+
+export const drawPoint = async (
```

Comment:
why change from `export async function`? is clearer than const

Reply:
it's just my habit using that style